### DISCUSSION TITLE
Possible 'bug' in the DynamicPolarizability.plotPolarizability() function

### DISCUSSION THREAD

**Discussion by eranreches-weizmann** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#121] [General]
Hi,

When using the DynamicPolarizability.plotPolarizability() function to plot the polarizability, the ylabel of the plot is 'Hz/V^2 cm^2' (see source code [here](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/_modules/arc/calculations_atom_single.html#DynamicPolarizability.plotPolarizability)), while the polarizability is actually given in units 'Hz/V^2 m^2' per both the documentation (see [here](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/generated/arc.calculations_atom_single.DynamicPolarizability.plotPolarizability.html#)) and my comparison to literature. Am I correct, or am I missing anything?

FYI, I am running ARC 3.2.1 on Python 3.10.

Thanks in advance for fixing the issue,
Eran

**Comment by nikolasibalic** [comment]
Hi Eran @eranreches-weizmann , thank you for noting this, well spotted!

I've fixed this (1ece0e160b4ba071545b396cc0517e745d2d175d) and new minor release v3.2.2 now includes correct label.

Thanks once again for taking time to report this!
